map/map_height_fitter/src/map_height_fitter.cpp

```diff
@@ -14,6 +14,8 @@
 
 #include "map_height_fitter/map_height_fitter.hpp"
 
+#include "memory"
```

Comment:
```suggestion
#include <memory>
```

Reply:
Thank you for the suggestion, I fixed it at 783191e .

I am not sure why, but this was done by clang-tidy's automatic fix...